docs/source/index.md

```diff
@@ -2,39 +2,30 @@
 
 <!--pytest.mark.skip-->
 ```python
-dataloader = torch.utils.data.DataLoader(dataset=ImageStreamingDataset(remote='s3://...'))
-```
-
-For additional details, please see our [Quick Start](getting_started/quick_start.md) and [User Guide](getting_started/user_guide.md).
+from torch.utils.data import DataLoader
+from streaming import StreamingDataset
 
-Streaming was originally developed as a part of MosaicML’s Composer training library and is a critical component of our efficient machine learning infrastructure.
+dataloader = DataLoader(dataset=StreamingDataset(remote='s3://...'))
+```
 
-## Installation
 
-```bash
-pip install mosaicml-streaming
-```
 
-## Key Benefits
 
-- High performance, accurate streaming of training data from cloud storage
-- Efficiently train anywhere, independent of training data location
-- Cloud-native, no persistent storage required
-- Enhanced data security—data exists ephemerally on training cluster
+## **🔑** Key Features
 
-## Features
+- **True Determinism**: Samples are in the same order regardless of the number of GPUs, nodes, or CPU workers. This makes it easier to reproduce and debug training runs and loss spikes and load a checkpoint trained on 64 GPUs and debug on 8 GPUs with reproducibility.
+- **Instant Mid-Epoch Resumption**: Resume training in seconds, not hours, in the middle of a long training run. Minimizing resumption latency can save thousands of dollars in egress fees and idle GPU compute time compared to existing solutions.
+- **High throughput**: Our MDS format cuts extraneous work to the bone, resulting in ultra-low sample latency and higher throughput compared to alternatives for workloads bottlenecked by the dataloader.
+- **Equal Convergence**: Model convergence from using StreamingDataset is just as good as using local disk, thanks to our shuffling algorithm. StreamingDataset shuffles across all samples assigned to a node, whereas alternative solutions only shuffle samples in a smaller pool (within a single process).
+- **Random access**: Access the data you need when you need it. Even if a sample isn’t downloaded yet, you can access `dataset[i]` to get sample `i`.
 
-- Drop-in replacement for {class}`torch.utils.data.IterableDataset` class.
-- Built-in support for popular open source datasets (e.g., ADE20K, C4, COCO, Enwiki, ImageNet, etc.).
-- Support for various image, structured and unstructured text formats.
-- Helper utilities to convert proprietary datasets to streaming format.
-- Streaming dataset compression (e.g., gzip, snappy, zstd, bz2, etc.).
```

Comment:
Are we just dropping these items?

Reply:
It is already being covered as part of [README.md](https://github.com/mosaicml/streaming/blob/main/README.md). We would like to keep it short and concise and couple of above points we will be covering as part of separate doc in detail.